docs/content/intro/protocol.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Download protocol"
+date: 2018-02-11T16:58:56-05:00
+---
+
+Athens builds on top of Go CLI which specifies a set of endpoints using which it communicates with external proxies providing modules. This set of endpoints we call _Download protocol_
```

Comment:
s/endpoints using which it/endpoints with which it/
s/_Download protocol_/the _Download Protocol_/